[Rule Tuning] Persistence via Update Orchestrator Service Hijack

imays11 opened the pull request:
<!--
Thank you for your interest in and contributing to Detection Rules!
There are a few simple things to check before submitting your pull request
that can help with the review process. You should delete these items
from your submission, but they are here to help bring them to your attention.
-->

## Issues
https://github.com/elastic/detection-rules/issues/2194

## Summary
- Changed query to exclude FPs for process.executables related to MoUsoCoreWorker.exe and OfficeC2RClient.exe. 
- Updated tags to include CVE


## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

terrancedejesus commented:
@imays11 I'd keep it EQL as you can still use wildcards. Below is a suggested update with it still being EQL.

```
process where event.type == "start" and
  process.parent.executable : "C:\\Windows\\System32\\svchost.exe" and
  process.parent.args : "UsoSvc" and
  not process.executable :
          ("?:\\ProgramData\\Microsoft\\Windows\\UUS\\Packages\\*\\amd64\\MoUsoCoreWorker.exe",
          "?:\\Windows\\System32\\UsoClient.exe",
          "?:\\Windows\\System32\\MusNotification.exe",
          "?:\\Windows\\System32\\MusNotificationUx.exe",
          "?:\\Windows\\System32\\MusNotifyIcon.exe",
          "?:\\Windows\\System32\\WerFault.exe",
          "?:\\Windows\\System32\\WerMgr.exe",
          "?:\\Windows\\UUS\\amd64\\MoUsoCoreWorker.exe",
          "?:\\Windows\\System32\\MoUsoCoreWorker.exe",
          "?:\\Windows\\UUS\\amd64\\UsoCoreWorker.exe",
          "?:\\Windows\\System32\\UsoCoreWorker.exe",
          "?:\\Program Files\\Common Files\\microsoft shared\\ClickToRun\\OfficeC2RClient.exe")
```

Tested this in our internal data collection and no matches for the last 2 months. I removed the MoUsoCoreWorker.exe additions just to make sure the query worked and had several thousand matches.


<img width="1743" alt="Screen Shot 2022-07-29 at 10 22 10 AM" src="https://user-images.githubusercontent.com/99630311/181780856-fddbc8ae-7940-4a9d-9c27-9a187b82d87c.png">

imays11 commented:
> @imays11 I'd keep it EQL as you can still use wildcards. Below is a suggested update with it still being EQL.
> 
> ```
> process where event.type == "start" and
>   process.parent.executable : "C:\\Windows\\System32\\svchost.exe" and
>   process.parent.args : "UsoSvc" and
>   not process.executable :
>           ("?:\\ProgramData\\Microsoft\\Windows\\UUS\\Packages\\*\\amd64\\MoUsoCoreWorker.exe",
>           "?:\\Windows\\System32\\UsoClient.exe",
>           "?:\\Windows\\System32\\MusNotification.exe",
>           "?:\\Windows\\System32\\MusNotificationUx.exe",
>           "?:\\Windows\\System32\\MusNotifyIcon.exe",
>           "?:\\Windows\\System32\\WerFault.exe",
>           "?:\\Windows\\System32\\WerMgr.exe",
>           "?:\\Windows\\UUS\\amd64\\MoUsoCoreWorker.exe",
>           "?:\\Windows\\System32\\MoUsoCoreWorker.exe",
>           "?:\\Windows\\UUS\\amd64\\UsoCoreWorker.exe",
>           "?:\\Windows\\System32\\UsoCoreWorker.exe",
>           "?:\\Program Files\\Common Files\\microsoft shared\\ClickToRun\\OfficeC2RClient.exe")
> ```
> 
> Tested this in our internal data collection and no matches for the last 2 months. I removed the MoUsoCoreWorker.exe additions just to make sure the query worked and had several thousand matches.
> 
> <img alt="Screen Shot 2022-07-29 at 10 22 10 AM" width="1743" src="https://user-images.githubusercontent.com/99630311/181780856-fddbc8ae-7940-4a9d-9c27-9a187b82d87c.png">

Thank you! I added those changes

DefSecSentinel reviewed (approved):
LGTM
